Makefile

```diff
@@ -0,0 +1,20 @@
+# Common variables
+NODE_MODULES=./node_modules
+PUBLIC_FOLDER=./public
+DIST_FOLDER=$(PUBLIC_FOLDER)/dist
```

Comment:
`article-tagging-prototype/dist`, again

Reply:
Again `article-tagging-prototype` folder should not have been in the repo at all; removed.